kafka-rest/src/test/java/io/confluent/kafkarest/controllers/TopicConfigManagerImplTest.java

```diff
@@ -201,6 +236,72 @@ public void listTopicConfigs_nonExistingCluster_throwsNotFound() throws Exceptio
     }
   }
 
+  @Test
+  public void listTopicConfigs_multipleExistingTopics_returnsConfigs() throws Exception {
+    expect(clusterManager.getCluster(CLUSTER_ID)).andReturn(completedFuture(Optional.of(CLUSTER)));
+    expect(
+        adminClient.describeConfigs(
+            eq(Arrays.asList(
+                new ConfigResource(ConfigResource.Type.TOPIC, TOPIC_NAME),
+                new ConfigResource(ConfigResource.Type.TOPIC, ALT_TOPIC_NAME)
+            )),
+            anyObject(DescribeConfigsOptions.class)))
+        .andReturn(describeConfigsResult);
+    expect(describeConfigsResult.all())
+        .andReturn(
+            KafkaFuture.completedFuture(new HashMap<ConfigResource, Config>() {{
+              put(new ConfigResource(ConfigResource.Type.TOPIC, TOPIC_NAME),CONFIG);
+              put(new ConfigResource(ConfigResource.Type.TOPIC, ALT_TOPIC_NAME),CONFIG);
+            }}));
+    replay(adminClient, clusterManager, describeConfigsResult);
+
+    Map<String, List<TopicConfig>> configs = topicConfigManager.listTopicConfigs(CLUSTER_ID,
+        Arrays.asList(TOPIC_NAME, ALT_TOPIC_NAME)).get();
+
+    assertEquals(
+        new HashSet<>(Arrays.asList(CONFIG_1, CONFIG_2, CONFIG_3)), new HashSet<>(configs.get(TOPIC_NAME)));
+    assertEquals(
+        new HashSet<>(Arrays.asList(ALT_TOPIC_CONFIG_1, ALT_TOPIC_CONFIG_2, ALT_TOPIC_CONFIG_3)),
+        new HashSet<>(configs.get(ALT_TOPIC_NAME)));
+  }
+
+  @Test
+  public void listTopicConfigs_multipleTopicsOneNonExistingTopic_throwsNotFound() throws Exception {
```

Comment:
nit: I can see that the existing tests use `throwsNotFound` even when they actually throw `UnknownTopicOrPartitionException`. I think that's a copy-paste omission, and we should use `throwsUnknownTopicOrPartition` here. I actually don't like `throwsNotFound` even where it's technically correct, because it alludes to returning an HTTP 404 response error, which is not relevant to manager unit tests.